README.md

```diff
@@ -38,6 +38,86 @@
 
 ---
 
+## Description
+
+`rio-tiler` was initialy designed to create [slippy map
+tiles](https://en.wikipedia.org/wiki/Tiled_web_map) from large raster data
+sources and render these tiles dynamically on a web map. With `rio-tiler` v2.0 we added many more methods to read
+data and metadata from any raster source supported by Rasterio/GDAL wherever
+they may be, including local files and via HTTP, AWS S3, Google Cloud Storage,
+etc.
+
+At the low level, `rio-tiler` is *just* a wrapper around [rasterio.vrt.WarpedVRT](https://github.com/mapbox/rasterio/blob/5b76d05fb374e64602166d6cd880c38424fad39b/rasterio/vrt.py#L15) class which can be usefull for doing reprojection and/or property overwriding (e.g nodata value).
+
+## Features
+
+- Read any dataset supported by GDAL/Rasterio
+
+```python
+from rio_tiler.io import COGReader
+
+with COGReader("my.tif") as image:
+    print(image.dataset)  # rasterio openned dataset
+    img = image.read()    # similar to rasterio.open("my.tif").read() but returns a rio_tiler.models.ImageData object
+```
```

Comment:
I think having some code in addition to the `feature` is a nice way to tell the user what the feature really is 🤷‍♂️